packages/react-native-win32/src/Libraries/Components/View/ViewWin32.Props.ts

```diff
@@ -223,4 +223,5 @@ export interface IViewWin32Props extends Omit<RN.ViewProps, ViewWin32OmitTypes>,
    * Provides a screentip to be used on hover of the view
    */
   tooltip?: string;
+  cursor?: string;
```

Comment:
Can we more specific on the valid values here?  -- I assume there is basically only a few options, its not really any string.

Reply:
I added a type Cursor with specific acceptable string values